### DISCUSSION TITLE
NextJS 13 Server Component / Page

### DISCUSSION THREAD

**Discussion by thisisrii** [vinissimus/next-translate#1030] [Q&A]
I'm trying to use ` const { t } = useTranslation('common')` in a server component / page but I always get the following error

<img width="744" alt="image" src="https://user-images.githubusercontent.com/8911615/232873716-1003a04a-58dd-4583-a497-8d46d3565fef.png">

I'm using version 13.3.0 of NextJS and 2.0.5 of next-translate and next-translate-plugin.

Please let me know if there's something else I can do or use an alternative, when looking at the example it seems like it should just work.

**Comment by aralroca** [comment]
How is your project? Are you using `pages` and `app` folder together? Please produce a reproducible example

**Reply by thisisrii** [reply]
I've forked the example and updated NextJS and next-translate and the plugin to the latest version and it doesn't seem to work.

https://codesandbox.io/p/sandbox/next-translate-app-dir-forked-joc7hq
